content/english/sql_workshop/_index.md

```diff
@@ -8,6 +8,7 @@ draft: false
 icon: "fas fa-database"
 weight: 130
 ---
+{{< youtube oFfaxcxBkUQ >}}

```

Comment:
I learned something new?????

This actually makes our life so much easier versus entering the entire embed code....